Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml

```diff
@@ -10,9 +10,9 @@
 	<ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
 	<ECSchemaReference name="Analytical" version="01.00.01" alias="anlyt" />
 	<ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU"/>
-	<ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru"/>
-	<ECSchemaReference name="NetworkTopology" version="01.00.00" alias="net"/>
-  <ECSchemaReference name="StormSewerPhysical" version="01.00.00" alias="stmswrphys"/>
+	<ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru"/>
+	<ECSchemaReference name="NetworkTopology" version="01.00.01" alias="net"/>
+  <ECSchemaReference name="StormSewerPhysical" version="01.00.01" alias="stmswrphys"/>
 
```

Comment:
This comment applies to line 6 (github doesn't let me add it up there): The WIP copy of the schema should be incremented when doing a release - e.g. 1.0.1.

Reply:
bumped the version to 01.00.01